### ISSUE TITLE
Using Receive-WinSCPItem with -Remove Param on Directory thows exception

### ISSUE THREAD

**Issue by dotps1** [dotps1/WinSCP#7]
After establishing a valid Session, using:
```PowerShell
$so = New-WinSCPSessionOptions -HostName host -UserName user -Password word -Protocol Ftp
$s = Open-WinSCPSession -SessionOptions $so
Receive-WinSCPSession -WinSCPSession $s -RemotePath "./Folder" -LocalPath "C:\Folder" -Remove
```
Throws an exception:
```
Exception calling "GetFiles" with "3" argument(s): "Operation is not valid due to the current state of the object." 
```

I have opened an issue on WinSCPs site:
http://winscp.net/forum/viewtopic.php?p=55303#55303

**Comment by dotps1** [comment]
This is a confirmed bug: http://winscp.net/tracker/show_bug.cgi?id=1299
It was a bug when deleting an empty directory after downloading it.

Will be fixed in version 5.7.2.

**Comment by dotps1** [comment]
Resolved with new version of WinSCP Assembly.
